feature-libs/user/profile/styles/my-account-v2/_my-account-v2-password.scss

```diff
@@ -0,0 +1,56 @@
+%cx-my-account-v2-password {
+  .myaccount-password-header {
+    @include type('3');
+    padding-bottom: 1.5rem;
+  }
+
+  .myaccount-password-button {
+    font-weight: var(--cx-font-weight-normal);
+  }
+
+  .myaccount-password-button-cancel {
+    color: var(--cx-color-primary);
+  }
+
+  .myaccount-password-label {
+    @include type();
+    min-width: 7.5rem;
+    padding: 0 1.2rem 0 0;
```

Comment:
Substitue `padding-left` and `padding-right` for  `padding-inline-start` and `padding-inline-start`. We need these for directionality to work.
`padding-top: 0` 
`padding-inline-end: 1.2rem`
`padding-bottom: 0`
`padding-inline-start: 0`